### PR TITLE
docs: add PHP sdk docs

### PR DESCRIPTION
**Pull request by darcyYe**
## Summary

add PHP SDK docs
localhost_3000_next_sdk_php_ (1)

### CODE REVIEW

File: docs/sdk/php/README.mdx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+### Implement sign-in route
+
+<ImplementSignInRoute />
+
+### Implement the callback route
+
+<ImplementCallbackRoute />
+
+### Implement the home page
```
Comment (wangsijie) [inline docs/sdk/php/README.mdx:47]: In other guides, the "home" and "callback" section is the sub-section of "Implement sign-in".

Comment (darcyYe) [reply docs/sdk/php/README.mdx]: Updated.

**Comment by xiaoyijun** [comment]
Need to add a link to the sample repo.

**Review by xiaoyijun — COMMENTED** [review]
Need to add a link to the sample repo.

**Review by xiaoyijun — APPROVED** [review]

**Pull request merged**